templates/hardwarelab/event_manager.html

```diff
@@ -0,0 +1,192 @@
+{% extends "layout.html" %}
+{% block content %}
+<script>
+$(document).ready(function(){
+    $(".create-rental").on("click", function() {
+      $rental_id = $(this).siblings("#rental-id").attr("value");
```

Comment:
do 

var $rental_id, and not $rental_id = ....

If you don't add var, you are adding to the global namespace. 
Also adding a $ in front of your variable means it's a jquery object. In this case, these values are strings , not jquery elements. This can be pretty confusing